src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2496,17 +2497,10 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
                   grainc_to_seed(p) = t1 * min(-cropseedc_deficit(p), grainc(p))
                   grainn_to_seed(p) = t1 * min(-cropseedn_deficit(p), grainn(p))
                   ! Send the remaining grain to the food product pool
-                  
-                  ! Y. Cheng, because I calculated grainc_to_food and grainn_to_food earlier
-                  if (ivt(p)==nmiscanthus .or. ivt(p)==nirrig_miscanthus .or. ivt(p)==nswitchgrass .or. ivt(p)==nirrig_switchgrass) then
-                  	grainc_to_food(p) = grainc_to_food(p) + t1 * grainc(p)  + cpool_to_grainc(p) - grainc_to_seed(p)
-                  	grainn_to_food(p) = grainn_to_food(p) + t1 * grainn(p)  + npool_to_grainn(p) - grainn_to_seed(p)
-				  else
-				  	grainc_to_food(p) = t1 * grainc(p)  + cpool_to_grainc(p) - grainc_to_seed(p)
-                  	grainn_to_food(p) = t1 * grainn(p)  + npool_to_grainn(p) - grainn_to_seed(p)
-				  end if
-
-                  livestemc_to_litter(p) = t1 * livestemc(p)  + cpool_to_livestemc(p)
+				  grainc_to_food(p) = t1 * grainc(p)  + cpool_to_grainc(p) - grainc_to_seed(p)
+                  grainn_to_food(p) = t1 * grainn(p)  + npool_to_grainn(p) - grainn_to_seed(p)
+
+                  livestemc_to_litter(p) = t1 * livestemc(p)  + cpool_to_livestemc(p)                 
```

Comment:
I commented on this on the duplicate (?) F90 routine, but I'll also add it here: Do you only harvest a fraction of the leaf biomass, or do you also want to add the stem biomass?